JHipster-Developer-Association-CLA.md

```diff
@@ -0,0 +1,81 @@
+### JHipster Developers Association Individual Contributor License Agreement
+
+Thank you for your interest in contributing to open source software projects (**"Projects"**) 
+made available by [JHipster Developers Association](https://www.jhipster.tech/association/) 
+or its affiliates (the **"Association"**). This Individual Contributor License Agreement (**"CLA"**) sets out the 
+terms governing any source code, object code, bug fixes, configuration changes, tools, specifications, 
+documentation, data, materials, feedback, information or other works of authorship that you submit or 
+have submitted, in any form and in any manner, to the Association in respect of any of the Projects 
+(collectively **“Contributions”**). If you have any questions respecting this Agreement, 
+please contact **info@jhipster.tech**
```

Comment:
@jdubois is the email id good?

Reply:
No. Best solution would be a Google Groups so we all receive it. I will create one.

Reply:
Will anyone be able to send emails to those group accounts? can't we set up a forwarding rule from the .tech email to a google group or something?

Reply:
I've many issues with forwarding rules from jhipster.tech, I'm guessing this is because of our registrar which isn't that good for emails... For Google Groups, and this is what I did for the association: I do a private group, but anybody can send emails to it. We might as well use the one of the association, btw.

Reply:
ok let me know when you create one and I'll update it

Reply:
How about jhipster-developers-association@googlegroups.com  ? This is for the board of the association (normal members don't see them), but that looks good for something like this (maybe if you have an issue, you don't want everybody to know about it)

Reply:
+1 for jhipster-developers-association@googlegroups.com

Reply:
if external person can send mail to jhipster-developers-association@googlegroups.com, it should be ok !

Reply:
Done